source/fundamentals/crud/read-operations/retrieve.txt

```diff
@@ -0,0 +1,242 @@
+.. _charp-retrieve:
+
+=============
+Retrieve Data
+=============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to retrieve data from your MongoDB
+collections using the ``Find()`` method.
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide use the ``sample_restaurants.restaurants`` collection
+from the :atlas:`sample datasets </sample-data>` provided by Atlas. See the 
+:ref:`<csharp-quickstart>` to learn how to create a free MongoDB cluster and 
+load this sample data.
+
+.. literalinclude:: ../../../includes/code-examples/Restaurant.cs
+   :language: csharp
+   :copyable:
+   :dedent:
+
+.. _csharp-retrieve-find:
+
+Find Operation
+--------------
+
+You can use the ``Find()`` method to retrieve documents from a collection. 
+The ``Find()`` method takes a **query filter**, that specifies your
+search criteria, and returns all matching documents as an ``IFindFluent`` interface.
```

Comment:
I don't think pointing out the IFindFluent interface is all that important to the rest of the documentation on this page so I removed it. I think it will be better to describe it more in other pages we have planned in the future. 

I added a little more detail to describe the query filter. Let me know how you feel about the new wording as it's not exactly what you suggested. Query Filters can be made in numerous ways and we are writing a page to detail all of them so I just linked to that (not yet made) page.

Reply:
Looks good!